### DISCUSSION TITLE
play.chakra-ui.com is broken

### DISCUSSION THREAD

**Discussion by Ploppy3** [chakra-ui/chakra-ui#8874] [General] [answered]
### Description

play.chakra-ui.com is broken

### Link to Reproduction

https://play.chakra-ui.com

### Steps to reproduce

2. Click on '...'
3. Scroll down to '...'

### Chakra UI Version

1.0.0

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

**Comment by segunadebayo** [comment]
Not broken for me. Please share a screenshot

**Reply by segunadebayo** [reply, marked as answer]
It should work correctly now.

**Comment by Ploppy3** [comment]
Somehow it has been fixed. It was probably an issue on codesandbox side

**Comment by apexskier** [comment]
This is broken for me as well at https://play.chakra-ui.com/. I'm seeing errors that look related to the v3 update (it looks like the playground is pegged to `latest` but is using v2 apis).

![Chakra Play - An interactive playground for Chakra UI - Safari - 2025-01-06 at 17 09 53@2x](https://github.com/user-attachments/assets/4560eb5e-47cb-4d75-ad8a-39cc75d9a184)

**Reply by apexskier** [reply]
I see similar issues with the stackblitz linked in https://github.com/chakra-ui/chakra-ui/blob/4eca0a6bb5f3db8e4edc0b875e0f22f4c46519d9/.github/ISSUE_TEMPLATE/bug_report.yml#L31.

**Reply by apexskier** [reply]
Here's a forked code sandbox that works with Chakra 3 - https://codesandbox.io/p/sandbox/rough-mountain-6pqhrc?file=%2FApp.tsx%3A9%2C1-25%2C1
